src/lib/udev/google_nvme_id

```diff
@@ -58,7 +58,7 @@ function err() {
 #######################################
 function get_namespace_device_name() {
   local nvme_json
-  nvme_json="$("$nvme_cli_bin" id-ns -b "$1" | xxd -p -seek 384 | xxd -p -r)"
+  nvme_json="$("$nvme_cli_bin" id-ns -b "$1" | dd bs=1 skip=384 2>/dev/null | od -x)"
```

Comment:
Do we need `od -x` here. This was just for visualization?

Reply:
You are right! Thanks @travier fixed.